pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -153,7 +157,7 @@ func attributesEnforcedInDynamicMetadataMatcher(k string) bool {
 		attrRequestPresenter:
 		return true
 	}
-	return strings.HasPrefix(k, attrRequestClaims)
+	return strings.HasPrefix(k, attrRequestClaims) || strings.HasPrefix(k, authDerivedClaims)
```

Comment:
Same comments. The derived claims should not be user facing. Why do we still check it here?

Reply:
This is an older code I forgot to remove. It has been removed.